transformedTests/readme.md

```diff
@@ -0,0 +1,13 @@
+Tests, for which the code has type information transformed in. This way, it's much easier to write mass tests for the validator.
+i.e.
+````typescript
+class A { // This class will be enhanced with type info
+    foo() { return true ? 123 : 'foo'; }
+}
+expect( reflect(A).getMethod('foo').returnType.isUnion() ).to.be.true
+````
+
+Currently, it uses the published typescript-rtti package from npm.
+
+TODO: Try to set it up as a monorepo (currently this fails because the code gets compiled to ../dist/transfomer and we get an error during the tspc run if we point the transformer to there. Propably there are some parent files missing. 
+  Sub-TODO: Set up the whole project to compile in-place (at least the dist dir) which would imho be the best for debugging. think of source maps and other stuff that have all these pointing back and forth issues. Or at least make the package.json match the published one as close as possible.
```

Comment:
What do you mean by this? Do you mean having `.js` next to `.ts`? I don't think we need to do that, if you just use `"typescript-rtti": ".."` in your package.json, you'll get the `src` and `dist` folders exactly where they need to be for sourcemaps to work (since the sourcemaps in `dist` refer to `../src/*`).

Reply:
> Do you mean having .js next to .ts
yes.

OK, if it works currently, then we should leave it.